### PR TITLE
CFY-5463 fix resource download for rollback

### PR DESCRIPTION
**Pull request by adamlavie**

### CODE REVIEW

File: components/utils.py [hunk 1]
```diff
@@ -928,6 +928,16 @@ def create(self, source, destination, service_name, user_resource=False,
                 self._set_resources_json(resources_props, service_name)
         return local_resource_path, destination
 
+    @staticmethod
```
Comment (nir0s) [inline components/utils.py]:
```python
return not os.path.isfile(local_res_path) or (is_render and is_upgrade) or False
```

?

File: components/utils.py [hunk 2]
```diff
@@ -928,6 +928,16 @@ def create(self, source, destination, service_name, user_resource=False,
...
+    def _is_download_required(local_res_path, is_render):
```
Comment (nir0s) [inline components/utils.py]: why ambiguous variable name? why not `local_resource_path`?

**Comment by nir0s** [comment]
Can you explain the reasoning behind this please?

File: components/utils.py (hunk 1 above)
Comment (adamlavie) [inline components/utils.py]: Simply for readability

**Pull request merged**